packages/flight-website/app/components/ds-icon-grid.hbs

```diff
@@ -14,7 +14,6 @@
         id="ds-input-search"
         class="ds-input-search"
         placeholder="e.g. arrow"
-        autofocus={{false}}
```

Comment:
this ran afoul of a newly enable rule:

```
$ ember-template-lint .
app/components/ds-icon-grid.hbs
  17:8  error  No autofocus attribute allowed, as it reduces accessibility by moving users to an element without warning and context  no-autofocus-attribute
```

Reply:
I'll submit an issue for this; we're saying no autofocus which is the same thing

Reply:
Ahh, that make sense. So deleting this isn't necessarily a problem but it _should_ be able to stay without tripping up the linter.